src/script/interpreter.cpp

```diff
@@ -1319,6 +1345,76 @@ template void PrecomputedTransactionData::Init(const CMutableTransaction& txTo,
 template PrecomputedTransactionData::PrecomputedTransactionData(const CTransaction& txTo);
 template PrecomputedTransactionData::PrecomputedTransactionData(const CMutableTransaction& txTo);
 
+static const CHashWriter HasherTapSighash = TaggedHash("TapSighash");
```

Comment:
Assuming this is in reply to https://github.com/bitcoin/bitcoin/pull/17977#discussion_r459531548
The string "TapSighash" itself, not the variable name, is consensus critical, because it gets hashed into transactions. So the code style guidelines aren't relevant.

Reply:
Changed the variable name.